src/ex3/README.md

```diff
@@ -0,0 +1,46 @@
+# Exercise 2 - In depth JS, Async, & MVC
```

Comment:
This isn't the right readme for this exercise, should put the relevant one so you'll remember the task definition and its requirements in the future